pkg/monitortests/kubeapiserver/apiservergracefulrestart/monitortest.go

```diff
@@ -161,7 +182,7 @@ func interesting(interval monitorapi.Interval) (monitorapi.IntervalReason, bool)
 	reason := interval.Message.Reason
 	switch reason {
 	// openshift-apiserver still is using the old event name TerminationStart
-	case "ShutdownInitiated", "TerminationStart", "TerminationGracefulTerminationFinished":
+	case "ShutdownInitiated", "TerminationStart", "TerminationGracefulTerminationFinished", "LateConnections":
```

Comment:
we already have a test that checks for "LateConnections" event, here: https://github.com/openshift/origin/blob/254f1e216526315abb05d9cdea025609f97f8bbd/test/extended/apiserver/graceful_termination.go#L132

Reply:
yes, thanks for pointing out, i will remove the junit test failed bits, just keep the part where the apiserver shutdown interval is marked red with an error message.